lib/driver.js

```diff
@@ -510,6 +510,7 @@ export class XCUITestDriver extends BaseDriver {
     // at this point if there is no platformVersion, get it from the device
     if (!this.opts.platformVersion) {
       this.opts.platformVersion = await this.device.getPlatformVersion();
+      this.caps.platformVersion = this.opts.platformVersion;
```

Comment:
I assume it would be better to assign it after the normalization happens (e.g. line 525)

Reply:
You're absolutely right, thank you! I changed the location where it's set. :)